controllers/favorites.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
         .populate("recipe")
         .skip(skip)
         .limit(limit);
+      console.log(`   The contents of favoriteRecipes`);
+      favoriteRecipes.forEach(recipe => console.log(recipe));
```

Comment:
A single-line arrow function without curly braces technically returns a value. To be a bit more proper, this line should be:

`favoriteRecipes.forEach(recipe => { console.log(recipe); });`

This way we're telling the function to just "do this thing" and not return anything. `console.log` just returns `undefined` anyway but this is good practice for getting better acquainted with arrow functions and return values.

Reply:
oh word?  Thank you for this catch.